config/secrets.yml

```diff
@@ -14,15 +14,21 @@ development:
   secret_key_base: 66a171a98f1a2c22fd866e6475c995421cd725b9cd1728bcd73532a0e966c6ec361d7c61dc7638c000151ef5341bf418cf212407dfe3a90cb40e5a7138ee0aeb
   api_url: https://stapp.sendaguy.com/api
   host: 'http://localhost:3000'
+  apple_store_url: "https://itunes.apple.com/us/app/sendaguy-now/id1047570020"
```

Comment:
@ramesh-aconnexion can we remove this file from git repo? We can use ENV vars for this.